cmd/livepeer/livepeer.go

```diff
@@ -44,6 +44,17 @@ import (
 var (
 	ErrKeygen    = errors.New("ErrKeygen")
 	EthTxTimeout = 600 * time.Second
+
+	// redeemGas is the gas required to redeem a PM ticket
+	redeemGas = 100000
+	// txCostMultiplier is the multipler on the transaction cost to use for PM ticket faceValue
```

Comment:
Why do we need a multiplier here?

Reply:
In the original outline there is mention of a transaction cost overhead, represented by a float. This represents the overhead cost for redeeming a ticket. 

For reasons such as rounding errors with floats and having to use the big.Float library (for more details you should ask @yondonfu ) this is now represented by a proportion between two integers (`redeemGas` and `txCostMultiplier`)

redeemGas is currently at `100 000`
txCostMultiplier is `100`

This would mean a transaction cost overhead of 1% 

If we look at the `faceValue` method in `recipient.go` we'll see: 

	```
gasPrice := r.gpm.GasPrice()
	// txCost = redeemGas * gasPrice
	txCost := new(big.Int).Mul(big.NewInt(int64(r.cfg.RedeemGas)), gasPrice)

	// faceValue = txCost * txCostMultiplier
	faceValue := new(big.Int).Mul(txCost, big.NewInt(int64(r.cfg.TxCostMultiplier)))
```

So let's say we have a gasPrice of 40 wei (not realistic but just for exampel sake), we'd get a transaction cost of `4 million wei` 

This would put the faceValue at 100x that , or `400 million wei` 

Which remains a 1% transaction cost overhead.

tl;dr redeemGas with txMultiplier replaces a float representation for transaction cost overhead.

Reply:
Thanks for the really detailed explanation @kyriediculous !! Makes sense to me